plsync/mlabconfig.py

```diff
@@ -8,6 +8,13 @@
 import sys
 import time
 
+ZONE_TTL = 60 * 5
+ZONE_MIN_TTL = 60 * 5
+ZONE_REFRESH = 60 * 5
```

Comment:
I think it's safe to say that all modern named implementations probably implement a notification mechanism so that a master can notify slaves of updates rather than slaves having to constantly poll.  I believe the refresh/retry values are a failover to notifications, in effect. 

Having the zone expiry at 7 days makes sense too, and I think this is a good value in the case that we have unexpected problems with our own master DNS server.